include/engine/plugins/trip.hpp

```diff
@@ -145,7 +137,7 @@ template <class DataFacadeT> class RoundTripPlugin final : public BasePlugin
             // whole graph is one scc
             std::vector<NodeID> location_ids(number_of_locations);
             std::iota(std::begin(location_ids), std::end(location_ids), 0);
-            return SCC_Component(std::move(location_ids));
+            return SCC_Component(std::move(location_ids), std::vector<size_t> {{0, location_ids.size()}});
```

Comment:
Given that the cases still fail, I remember that even constructing like this did not work in the prototype.
I had to explicitly construct the vector prior to the return statement and move it in.
`std::vector<size_t> range = {0.location_ids.size()};`
`return SCC_Component(std::move(location_ids), std::move(range));`